doc/release/1.14.0-notes.rst

```diff
@@ -307,6 +307,25 @@ In summary, the major changes are:
 * User-defined ``dtypes`` (subclasses of ``np.generic``) now need to
   implement ``__str__`` and ``__repr__``.
 
+You may want to do something like:
+```
+# FIXME remove next 3 lines once 1.14 is minimum NumPy requirement
+# and fix printing doctests to use new format.
+from distutils.version import LooseVersion
+if LooseVersion(np.__version__) >  LooseVersion("1.13.3"):
```

Comment:
This should be recommending `LooseVersion(np.__version__) >= LooseVersion("1.14.0")` instead.

Or maybe just

```
try:
    np.set_printoptions(legacy="1.13")
except TypeError:
    pass

Reply:
That is what I ended up using for networkx (https://github.com/networkx/networkx/pull/2810).  I modified the commit accordingly and switched to rst.  I am not sure where this should go (assuming it is desirable), so I will leave it here until someone has a better suggestion.